### ISSUE TITLE
Pay Button has NO TEXT!!!

### ISSUE THREAD

**Issue by webdeb** [paypal/paypal-checkout-components#1447]
### 🐞 Describe the Bug

Debit/Credit Dialog -> "PAY Button" has no text

![Bildschirmfoto 2020-10-07 um 14 12 53](https://user-images.githubusercontent.com/14992140/95329733-b9851780-08a7-11eb-8255-358e20c2b050.png)

### Affected browsers

What browser(s) are affected?

- Chrome

### ➕ Additional Context

Language: German

**Comment by nick4414** [comment]
I've already reported the issue to the paypal development team and they are working on it.

**Comment by gregjopa** [comment]
I believe this has been fixed. I just tested using the German locale (locale=de_DE) and I see text for the Pay Now button:

<img width="389" alt="Screen Shot 2020-11-23 at 3 25 17 PM" src="https://user-images.githubusercontent.com/534034/100017219-2e50da00-2da0-11eb-8036-037902b85b60.png">

I'm going to close this one out. @webdeb and @nick4414, if you are still seeing this issue we can re-open. Just let us know.
